Components/TransactionCard.js

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function TransactionCard({ transaction }) {
+  let iconImageSrc = "";
+  if (transaction.category === "Groceries") {
+    iconImageSrc = "salad.svg";
+  }
+  if (transaction.category === "Rent") {
+    iconImageSrc = "house.svg";
+  }
+  if (transaction.category === "Salary") {
+    iconImageSrc = "hand-coins.svg";
+  }
+  if (transaction.category === "Insurance") {
+    iconImageSrc = "umbrella.svg";
+  }
+  if (transaction.category === "Entertainment") {
+    iconImageSrc = "clapperboard.svg";
+  }
+  if (transaction.category === "Restaurants") {
+    iconImageSrc = "utensils.svg";
+  }
+  if (transaction.category === "Investment") {
+    iconImageSrc = "chart-line.svg";
+  }
+  if (transaction.category === "Transportation") {
+    iconImageSrc = "car.svg";
+  }
+  if (transaction.category === "Education") {
+    iconImageSrc = "graduation-cap.svg";
+  }
+  if (transaction.category === "Savings") {
+    iconImageSrc = "circle-dollar-sign.svg";
+  }
+
+  return (
+    <>
+      {/* Hier wird die Catergorys abgefragt und wenn eine true ist wird das dazugehörige Symbol
+    eingefügt */}
+      <StyledTransactionCard type={transaction.type}>
+        <StyledIcon src={iconImageSrc} alt="bild" width="25" height="25" />
```

Comment:
```suggestion
        <StyledIcon src={iconImageSrc} alt={`${transaction.category}-icon`} width="25" height="25" />
```

'bild' is not a proper description 😊

Reply:
changed as per request